apps/web/src/pages/SupportPage/components/SupportForm/SupportForm.tsx

```diff
@@ -102,13 +104,16 @@ const SupportForm = ({
   currentUser,
 }: SupportFormProps) => {
   const intl = useIntl();
+  const location = useLocation();
+  const previousUrl = location?.state.referrer ?? document.referrer;
```

Comment:
I'm getting an error at this line when I go from `/en/privacy-notice` to Contact Us page

Reply:
Strange, I don't see any error. Maybe your environment needs rebuilding from switching branches?

Try adding `console.log(previousUrl);` on line 109 after the `previousUrl` is defined and building again `npm run dev`. Does anything come back in your browser console?

Reply:
Good catch @tristan-orourke, my bad. Should be fixed in 51dd01517fa80c34fa8f060bfdebd0b5c9bce049.